modules/homefinder/widgets/class-affiliate-search-widget.php

```diff
@@ -0,0 +1,208 @@
+<?php
+/**
+ * HomeFinder Affiliate Search Widget (http://www.homefinder.com/widgets/)
+ *
+ * @package RE-PRO
+ */
+
+	/* Exit if accessed directly. */
+if ( ! defined( 'ABSPATH' ) ) { exit; }
+
+/**
+ * HomeFinderAffiliateSearch class.
+ *
+ * @extends WP_Widget
+ */
+class HomeFinderAffiliateSearch extends WP_Widget {
+
+	/**
+	 * __construct function.
+	 *
+	 * @access public
+	 * @return void
+	 */
+	public function __construct() {
+
+		parent::__construct(
+			'homefinder_affiliate_search',
+			__( 'HomeFinder Affiliate Search', 're-pro' ),
+			array(
+				'description' => __( 'Display an affiliate search box from HomeFinder.com', 're-pro' ),
+				'classname'   => 're-pro re-pro-widget homefinder-widget homefinder-affiliate-search',
+				'customize_selective_refresh' => true,
+			)
+		);
+	}
+
+	/**
+	 * Widget function.
+	 *
+	 * @access public
+	 * @param mixed $args Arguments.
+	 * @param mixed $instance Instance.
+	 */
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->